app/server.js

```diff
@@ -27,6 +29,18 @@ export async function createHttpServer() {
   app.use(compression())
   app.use(bodyParser.json())
 
+  client.register.clear()
+  app.use(
+    promBundle({
+      includePath: true,
+      promClient: {
+        collectDefaultMetrics: {
+          prefix: 'dscp_identity_service_',
```

Comment:
can we just say identity_service_ to be inline with ipfs

Reply:
https://github.com/digicatapult/dscp-ipfs/blob/main/app/server.js#L36

Reply:
No worries, I was told to put `dscp_identity_service_`, I'll change it

Reply:
No leave it as `dscp-identity-service`, the ipfs one needs to be changed.